### ISSUE TITLE
Documentation for TeachingBubble is incomplete

### ISSUE THREAD

**Issue by hasridha** [microsoft/fluentui#17178]
1. Open https://developer.microsoft.com/en-us/fluentui#/controls/web/teachingbubble
2. Try to find any documentation for a property/etc to set the aria-label for the bubble close button

**Result** - A keen observer may find a single reference to IAccessiblePopupProps on the page - but it's in a very small font, extremely hidden away and isn't even clear if it's a hyperlink/leads to more documentation.
![image](https://user-images.githubusercontent.com/35580499/109211926-c4135100-7763-11eb-8a31-fec1253f671f.png)

**Expect** - all the props under IAccessiblePopupProps should be included in the set of interfaces documented on the link above - just like ITeachingBubbleSubComponentStyles etc is already listed and searchable on the page itself directly (and in every Fluent UI control where it is used)

**Comment by joschect** [comment]
@hasridha Thanks for the suggestion! I think that making this more visible is a good idea. We do this for all of our controls though and it's not just the TeachingBubble. You can see the same type of link for Dropdown (see screenshot). @ecraig12345 might know more about if we're planning to update the websites styling.

![image](https://user-images.githubusercontent.com/20801176/109601708-32457400-7ad4-11eb-839d-85625ee07a8f.png)

**Comment by ecraig12345** [comment]
`IAccessiblePopupProps` should already be a link, it's just not showing up as one due to some global styling conflicts that we'll finally be able to get rid of soon (@smhigley is working on this).

For a more obvious way of surfacing inherited props, we have a tracking issue #12043, but we probably won't get to that any time soon.

**Comment by gouttierre** [comment]
@smhigley moving this to backlog - as you are aware of it.

**Comment by ecraig12345** [comment]
As of today we finally fixed the issue that was making the link invisible:
![image](https://user-images.githubusercontent.com/5864305/110838362-38b3b880-8257-11eb-8fd7-2efbb4b17837.png)

So I'm going to close this one in favor of the general tracking issue about better ways to surface inherited props (#12043).
